net.infotexture.dita-bootstrap.json

```diff
@@ -154,5 +154,25 @@
     ],
     "url": "https://github.com/infotexture/dita-bootstrap/archive/5.3.2.zip",
     "cksum": "7d29ef6da445ebb0c7dcec84d191f2c118ad9fa15ea77c92b7f274550de5aca8"
+  },
+  {
+    "name": "net.infotexture.dita-bootstrap",
+    "description": "Styled HTML5 output with an extensible Bootstrap template",
+    "keywords": ["HTML", "HTML5", "Bootstrap", "Bootstrap 5", "Responsive"],
+    "homepage": "https://infotexture.github.io/dita-bootstrap",
+    "vers": "5.3.3",
+    "license": "Apache-2.0",
+    "deps": [
+      {
+        "name": "org.dita.base",
+        "req": ">=3.7.0"
+      },
+      {
+        "name": "fox.jason.extend.css",
+        "req": ">=1.0.0"
```

Comment:
@jason-fox There are newer versions of this plug-in, but should we **require** something newer, or will it still work with 1.0.0?

Reply:
If guess it should work, but 1.1.0 had some useful bug fixes and I would set a minimum of 1.2.0 if I were you. 
Additionally I should make a new release 1.2.2 myself - https://github.com/jason-fox/fox.jason.extend.css/issues/7 **tangentally** relates to DITA Bootstrap - or rather overriding the `common-bootstrap.css`

Reply:
If you like, this can remain in DRAFT state until you publish 1.2.2 and we can require that instead.

In the meantime, I'll push new commits here for the Lunr & Sass plug-ins.

Reply:
`fox.jason.extend.css` 1.2.2 is bundled into #141